packages/internals/src/migrateEngineCommands.ts

```diff
@@ -95,10 +114,15 @@ export async function canConnectToDatabase(
           message: error.fields.message,
         }
       } else {
-        throw new Error(`Migration engine error:\n${logs.map((log) => log.fields.message).join('\n')}`)
+        const shortMessage = redactFailedCommand(e.shortMessage)
```

Comment:
`e.shortMessage` is, for instance, `Command failed with exit code 127: /home/davi/Projects/quarta-casa/Repos/fisher-backend/node_modules/@prisma/engines/migration-engine-debian-openssl-1.0.x cli --datasource postgresql://postgres:root@localhost:5432/postgres?schema=public can-connect-to-database` (as can be seen in https://github.com/prisma/prisma/issues/13165)

Reply:
After `redactFailedCommand`, `shortMessage` becomes `'Command failed with exit code 127: /home/davi/Projects/quarta-casa/Repos/fisher-backend/node_modules/@prisma/engines/migration-engine-debian-openssl-1.0.x cli --datasource <REDACTED> can-connect-to-database'`